packages/aragon-cli/src/cli.js

```diff
@@ -100,5 +100,15 @@ const cli = yargs
 
 AragonReporter.configure(cli)
 cli.middleware(MIDDLEWARES)
+
+// Runs if command.handler is successful
+cli.onFinishCommand(() => {
+  process.exit()
+})
+// Runs if command.handler throws
+cli.fail((...args) => {
+  AragonReporter.errorHandler(...args)
```

Comment:
Shouldn't we use process.exit(1) here?
(And process.exit(0) in line 105?)

Reply:
1) It's not necessary to specify `0`, according to the docs, the default is 0 https://nodejs.org/api/process.html#process_process_exit_code
2) The `AragonReporter.errorHandler` handles the error reporting and calls process.exit(1) when necessary